src/CSS.cpp

```diff
@@ -13,36 +13,33 @@ namespace CSS
     {
         sProvider = gtk_css_provider_new();
 
-        struct CSSDir
-        {
-            std::string xdgEnv;
-            std::string fallbackPath;
-            std::string relPath;
-        };
-        std::string home = getenv("HOME");
-        std::array<CSSDir, 4> locations = {
-            CSSDir{"XDG_CONFIG_HOME", home + "/.config",      "/gBar/style.css"}, // Local config
-            CSSDir{"XDG_DATA_HOME",   home + "/.local/share", "/gBar/style.css"}, // local user install
-            CSSDir{"",                "/usr/local/share",     "/gBar/style.css"}, // local all install
-            CSSDir{"",                "/usr/share",           "/gBar/style.css"}, // package manager all install
-        };
+        std::vector<std::string> locations;
+        const char * env;
+
+        env = std::getenv("XDG_CONFIG_HOME");
+        ASSERT(env, "XDG_CONFIG_HOME is not set!");
+        locations.push_back(env);
```

Comment:
Missing fallback ("$HOME/.config") for XDG_CONFIG_HOME. Please include fallback before XDG_DATA_HOME.

Also missing XDG_DATA_HOME + fallback("$HOME/.local/share") here for local user installations.